platform/solr/solr-query/pom.xml

```diff
@@ -45,17 +45,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.0</minimum>
+                                            <minimum>0.73</minimum>
```

Comment:
I added a couple more tests to make sure every branch of the changed code is covered but there is still a pretty big gap in the numbers.